keps/sig-storage/20191117-immutable-ephemeral-volumes.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Immutable ephemeral volumes
```

Comment:
Hmm...I think the title can just be "Immutable Secrets and ConfigMaps"

The secrets and configmaps won't be mutable, whether they are consumed as ephemeral volumes.
(It's also slightly unclear what the scope of ephemeral volumes is)

Reply:
done